core/cmd.go

```diff
@@ -50,6 +50,19 @@ func (ch *CommandHandler) GetAppVersion() string {
 	return ch.cfg.AppVersion
 }
 
+func (ch *CommandHandler) GetTranslation(locale string) (*locales.Localization, error) {
+	lng, err := i18n.GetTranslation(locale)
+	if err != nil {
+		log.Println(err)
+		return nil, fmt.Errorf(`failed to get translation: %w`, err)
```

Comment:
Since GetTranslation only errors if none could be found, can return a StatusNotFound here